integration/docker/cgroups_test.go

```diff
@@ -135,6 +136,9 @@ var _ = Describe("Checking CPU cgroups in the host", func() {
 				expectedPeriod = "100000"
 				expectedCpuset = "1"
 
+				if runtime.GOARCH == "ppc64le" {
+					expectedCpuset = "8"
```

Comment:
Is this going to be reliable on every ppc64le system?

Reply:
Yes.